docs/DeveloperGuide.md

```diff
@@ -1,34 +1,630 @@
+[Appendix D: Instructions For Manual Testing](#appendix-d-instructions-for-manual-testing)
 
-### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
 
-## User Stories
+## 1. Introduction
+
+---
+
+**Greetings from Finux!**
+
+**Finux** is a CLI Style application that allows the user to make better financial
+decisions based on the information recorded in the application. If you’re familiar
+with the CLI command interface, you will enjoy the benefit of speeding up your
+finance management rather than using the traditional management system.
+
+**Finux** is written in Java 11 and uses the Object-Oriented Programming (OOP) paradigm.
+OOP enables us to produce organized and reusable codes, allowing efficient improvements
+and revisions in the future.
+
+### 1.1 Purpose
+
+This guide is for new and current developers of **Finux**. It describes the overall architecture
+design of **Finux** and describes the current implementation details of our main features
+with the thought process behind each of them. The current edition of this guide is meant for
+the `v2.1` release.
+
+### 1.2 Using this Guide
+
+Throughout the guide, you might encounter several icons, refer to the following consolidate
+list of icons:
+
+> 💡 It indicates that this is an important piece of information.\
+> ❗  It indicates that this is a warning and you should be aware of it while developing.\
+> 📝 It indicates that this is a tip and you may find it useful when using our application or guide.
+
+> 📝 Text that are blue like this [example(jumps to 1.2)](#12-using-this-guide), are clickable links
+> that will move you to the relevant section of the guide.
+
+## 2. Setting Up
+
+---
+
+### 2.1 Setting up the project in your computer
+
+Ensure that you have the following installed:
+* JDK 11
+* Intellij IDEA (Highly Recommended)
+
+Firstly, **fork** this repo and **clone** a copy into your computer.
+
+If you plan to use IntelliJ IDEA:
+1. **Ensure IntelliJ is configured to develop in JDK 11.**
+   1. If you are not at the welcome screen, click `File` > `Close Projects` to close any currently opened projects.
+   1. Click on `Configure` > `Structure for New Projects` > `Project Settings` > `Project`, ensure
+      the `Project SDK` is using **JDK 11**.
+1. **Import the project as a Gradle Project.**
+   1. Click on `Import Project` and locate the `build.gradle` file and select it. Click `OK`.
+   1. If prompted, choose to `Open as Project` (not `Open as File`).
+   1. Click `OK` to accept the default settings but do ensure that the selected version of `Gradle JVM`
+      matches the JDK being used for the project.
+   1. Wait for the importing process to finish (it could take a few minutes).
+1. **Verify the setup.**
+   1. After importing successfully, locate the `src/main/java/seedu.duke/Duke.java` file, right click it, 
+      and choose `Run...`. If the setup is correct, you should see the following:
+   ![Login_Page](img/LoginPageExampleOutput.png)
```

Comment:
Perhaps the unused space on the right side can be cropped out so that you can have more space to make the image bigger.

Reply:
Yes, thank you, will change it!